libs/pages/elewa/home/src/lib/activity-section/activity-section.component.scss

```diff
@@ -0,0 +1,29 @@
+.body{
+    background-color:white;
+    height: auto;
+    padding: 50px 150px;
+}
+.title{
+    font-size: 50px;
+    font-weight: var(--elewa-group-website-font-weight-subtitle);
+    font-family: var(--elewa-group-website-dmsans-bold);
+}
+p{
+    font-size: var(--elewa-group-website-font-size-title);
+    font-weight: var(--elewa-group-website-font-weight-subtitle);
+    font-family: var(--elewa-group-website-dmsans-regular);
+}
+.e-card{
+    display: grid;
+    grid-template-columns: auto auto auto;
```

Comment:
This entire section needs to be inside a components folder on the home module.

Please sync your fork